pkg/sink/sink.go

```diff
@@ -72,27 +72,39 @@ func (r Resource) HandleEvent(response http.ResponseWriter, request *http.Reques
 	log.Printf("EventListener: %s in Namespace: %s handling event (EventID: %s) with payload: %s and header: %v",
 		r.EventListenerName, r.EventListenerNamespace, eventID, string(event), request.Header)
 
+	result := make(chan bool, 10)
 	// Execute each Trigger
 	for _, trigger := range el.Spec.Triggers {
-		go r.executeTrigger(event, request, trigger, eventID)
+		go r.executeTrigger(event, request, trigger, eventID, result)
+	}
+	success := false
+	for i := 0; i < len(el.Spec.Triggers); i++ {
+		if <-result {
+			success = true
+		}
+	}
+	if !success {
+		response.WriteHeader(http.StatusInternalServerError)
```

Comment:
Are we sure we always want to return a 5xx if we fail to process any triggers? 
e.g. what if it is a event that the event listener is not interest in (e.g. push to a dev branch or something).
5xx usually implies that the server is having issues and retrying later should work which is not what we'd want here.

Reply:
Thanks for the reference.  I'll make changes to return 201 for the good case and 202 for no resources are created. 

About the 10 seconds, it should not take such long time to process the triggers.  We can document to reduce the number of triggers if it happens.

Reply:
Sounds good to me. My main concern is around the interceptor http requests hanging though I think we can fix that by setting low request timeout values on the http client we use.